iis-wf/iis-wf-transformers/src/test/resources/eu/dnetlib/iis/wf/transformers/metadataextraction/checksum/preprocessing/sampledataproducer/data/output_document_content_url.json

```diff
@@ -1 +1 @@
-{"id": "12345", "url": "http://someurl1.com", "mimeType": "pdf", "contentChecksum": "12345", "contentSizeKB": 0}
\ No newline at end of file
+{"id": "12345", "url": "http://someurl2.com", "mimeType": "pdf", "contentChecksum": "12345", "contentSizeKB": 0}
```

Comment:
Actually I looked at pig documentation of `LIMIT` and encounter on not good info:
`There is no guarantee which n tuples will be returned, and the tuples that are returned can change from one run to the next`
so we have to fix this test